mkdocs.yml

```diff
@@ -73,6 +73,7 @@ nav:
     - user-guide/passwords.md
     - user-guide/support.md
     - user-guide/betatest.md
+    - basic/cloud-kernel.md
```

Comment:
Move to adv-topics?

Reply:
ok

Reply:
Looks like you moved the file so this reference needs to be updated to match.